modules/tensor_mechanics/src/materials/ADComputeDilatationThermalExpansionEigenstrainBase.C

```diff
@@ -12,7 +12,8 @@
 InputParameters
 ADComputeDilatationThermalExpansionEigenstrainBase::validParams()
 {
-  return ADComputeThermalExpansionEigenstrainBase::validParams();
+  auto params = ADComputeThermalExpansionEigenstrainBase::validParams();
+  return params;
```

Comment:
What are we achieving with that extra declaration?

Reply:
ugh, nothing. while I was refactoring I moved a parameter here and later deleted it again.